src/deletepy/cli/main.py

```
@@ -0,0 +1,129 @@
+"""Click-based CLI entry point for DeletePy Auth0 User Management Tool."""
+
+import sys
+from pathlib import Path
+from typing import Optional
+
+import click
+
+from ..core.auth import get_access_token
```

Flake8 (7.2.0)

[error] '..core.auth.get_access_token' imported but unused

(F401)